### DISCUSSION TITLE
How to connect external led to ESP32-CAM?  (all pins taken)

### DISCUSSION THREAD

**Discussion by Sheugel** [jomjol/AI-on-the-edge-device#984] [Q&A] [answered]
Hi,
I purchased an ESP32-CAM on Amazon with the ESP32-S module and ESP32-CAM-MB:
[https://m.media-amazon.com/images/W/WEBP_402378-T2/images/I/71LGsWic1pS._AC_UL480_FMwebp_QL65_.jpg](url)
The 2 modules interconnect making so that all pins are taken.
I wonder how  I can connect a LED strip. Do I have to use jumpers? (without led strip, image quality is too bad)
Thanks!

**Comment by JensHoRi** [comment]
Good question. I only use the esp32 cam. There is in the wiki an explanation for external led.

don’t know for what you have the second module??

**Comment by Sheugel** [comment, marked as answer]
Hi, thanks for your answer. Second module provides power and ability to flash like a USB-UART connector (if I understand correctly).
If I use a USB-UART connector instead, you say I will be able to connect and power a led strip?
Thanks

**Reply by JensHoRi** [reply]
After flashing you could take it away. 

I use this cable (Amazon) 4pin PL2303 USB to UART TTL Cable Module RS232 Serial 4-pin PL2303HX Cable for power connection directly on the pins.

**Reply by caco3** [reply]
The `ESP32-CAM-MB` you have is just an additional PCB to provide an USB socket. I used it for initial flashing.

Afterwards i removed it and use the `5V` pin to power it from a cut USB cable.

I originally also thought I need an external LED due to the reflections. But then I build a diffuser out of "[Pergamentpapier](https://www.amazon.de/Wertpack-Backtrennpapier-Backpapier-beidseitig-silikonisiert/dp/B077BSLW7D)". I think in english its called "parchment paper".

**Reply by Sheugel** [reply]
Thanks to you both! I got it working with the 4pin cable. It allows me to power and send commands to the LED strip with the ESP32-CAM (ESP32-S module).
